contracts/InvestorActions.sol

```diff
@@ -58,7 +58,7 @@ contract InvestorActions is DestructibleModified {
   {
     var (ethTotalAllocation, ethPendingSubscription, sharesOwned, sharesPendingRedemption, ethPendingWithdrawal) = fund.getInvestor(_addr);
 
-    uint ethFilledAllocation = ethPendingSubscription.add(sharesToEth(sharesOwned));
+    uint ethFilledAllocation = ethPendingSubscription.add(fund.sharesToEth(sharesOwned));
```

Comment:
@mifeng I think later versions wouldn't it be better to keep helper functions in the upgradeable investorActions?  But low likelihood we'll have to update those basic functions.
So the fund would be the datastore (investor info, NAVs, usdeth, etc) and interface, helpers also live in upgradeable contracts.

Reply:
yep, that makes sense.  i'll make that change

Reply:
I'm going to merge this in as-is because putting the helpers into investorActions will create more complexity in terms of dependencies.  Also, the calcs in the helpers shouldn't change.